src/vs/editor/contrib/folding/browser/foldingRanges.ts

```diff
@@ -246,7 +246,7 @@ export class FoldingRegions {
 	}
 
 	public toFoldRange(index: number): FoldRange {
-		return <FoldRange>{
+		return {
```

Comment:
We can remove this assertion because the function has an explicit return type that will check the object